src/components/Hero.jsx

```diff
@@ -0,0 +1,161 @@
+import { PersonalizedText } from './shared/typography';
+import { Paragraph } from './shared/typography';
+
+const Hero = () => {
+	return (
+		<>
+			<div className='flex flex-col md:flex-row'>
+				<div className='w-[100%] md:w-1/2 '>
+					<div className='h-1/3 flex flex-col justify-end ml-2' style={{ color: '#FF4409' }}>

```

Comment:
What's the use setting height here?

Reply:
so as to divide into 3 sub divs under the left main div

Reply:
@aryab2003 need to use flex for that

Reply:
fixed https://github.com/dscnitrourkela/project-udon/pull/17/commits/3744a23efb73bcf7bc73ba221099c39ba27c9082